RELEASE_NOTES.md

```diff
@@ -8,15 +8,7 @@
 [//]: # ( ### Deprecated)
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
-## Added
-- Validation of PublicKey in CSE module if it's used standalone.
-- 'isReady' flag to ComponentState because some components might require some initialization time even if all the inputs are valid.
-- 'isValid' now checks both 'isInputValid' and 'isReady' to be true.
-
-## Fixed
-- Handle Intent results if DropInActivity got destroyed.
-- Queue API request if DropInService is not yet bound to DropInActivity
-
-
-
-
+## Changed
+- Updated Material Components dependency to 1.3.0.
+- The `merchantAccount` parameter in `GooglePayConfiguration.Builder` is now optional.
```

Comment:
Also add that they might need to update the code because of the new constructor, and that if not manually set, the Google environment will follow the Adyen environment automatically.